analyses/methylation-summary/README.md

```diff
@@ -0,0 +1,135 @@
+- Using `03-calculate-beta-quantiles.R` script and the methylation `Beta-values matrix`, calculate probe-level `quantiles` for each cancer type (cancer_group) within an OpenPedCan cohort (`results/methyl-probe-beta-quantiles.tsv.gz`). 
+- Using `04-beta-tpm-correlation.R` script and the methylation `Beta-values matrix`, calculate probe-level `correlations` between `RNA-Seq TPM-values` and `Beta-values` for each cancer type (cancer_group) within an OpenPedCan cohort for patients who have both datasets (`results/methyl-probe-beta-tpm-correlations.tsv.gz`). Correlations for cohorts will have independent scripts becuase of the differences in how samples with patients in both RNA-Seq and methylation data are determined. 
+- Summarize all results using `05-create-methylation-summary-table.R` and `06-methyly-summary-tsv2jsonl.py` scripts into a methylation summary table (`results/methyl-beta-values-summary.rds`, `results/methyl-beta-values-summary.tsv.gz` and `results/methyl-beta-values-summary.jsonl.gz`) that will be utilized with OPenPedCan plotting API and displayed on the NCI MTP portal with the following columns:
+    - **Gene_Symbol**: gene symbol
+    - **targetFromSourceId**: Ensemble ID
+    - **PMTL**: Is gene on PMTL (`Relevant Molecular Target`) 
+    - **Dataset**: OpenPedCan `cohort` i.e., TARGET
+    - **Disease**: OpenPedCan `cancer_group` from file i.e., Neuroblastoma
+    - **diseaseFromSourceMappedId**: EFO ID of OpenPedCan `cancer_group`
+    - **MONDO**: MONDO_ID of OpenPedCan `cancer_group`
+    - **RNA_Correlation**: array probe-level correlation between `methylation Beta-values` and `RNA-Seq TPM values`
+    - **Probe_ID**: `Illumina Infinium HumanMethylation` array probe ID for the CpG site
+    - **Chromosome**: chromosome for CpG site eg. chr1
+    - **Location**: genomic location of the CpG site
+    - **Beta_Q1**: array probe-level Beta Q1 quantile
+    - **Beta_Q2**: array probe-level Beta Q2 quantile
+    - **Beta_Median**: array probe-level Beta Q3 quantile
+    - **Beta_Q4**: array probe-level Beta Q4 quantile
+    - **Beta_Q5**: array probe-level Beta Q5 quantile
+    - **datatypeId**: Illumina_methylation_array
+    - **chop_uuid**: generate UUID
+    - **datasourceId**: chop_gene_level_methylation
+
+
+## General usage of scripts
+Analyses involving 850k arrays with large number of samples representing OPenPedCan cancer groups (as in the CBTN cohort) will require `~256gb` of memory to run successfully. In some instances the `system /tmp` is too small to hold temporary files generated during analysis by R scripts. Users are advised to create create a `./tmp` in the module directory then execute R script by prepending with TMP/TMPDIR environmental variable as illustrated in the wrapper module bash script, `run-methylation-summary.sh`.
+
+
+#### `Rrequired R packages`
```

Comment:
```suggestion
#### `Required R packages`
```

Reply:
This was resolved similar comment here: https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/165#discussion_r929048910